resources/istio/charts/security/templates/cleanup-secrets.yaml

```diff
@@ -80,6 +80,8 @@ spec:
   template:
     metadata:
       name: istio-cleanup-secrets
+      annotations:
```

Comment:
It is likely installed in istio-system namespace, this namespace does not have sidecar injection configured. Do we need this annotation here then?